.gitmodules

```diff
@@ -38,5 +38,5 @@
 	path = doc/doc-builder
 	url = https://github.com/ESMCI/doc-builder
 [submodule "components/cism"]
```

Comment:
This is wrong.  The url is https://github.com/jedwards4b/cism-wrapper, the branch is remove_manage_externals and the commit is 06f0a4592